packages/router/src/adapters/contract/contract.ts

```diff
@@ -209,16 +249,51 @@ export const prepare = async (
     },
   ]);
 
-  return await txService.sendTx(
-    {
-      to: nxtpContractAddress,
-      data: encodedData,
-      value: constants.Zero,
-      chainId,
-      from: wallet.address,
-    },
-    requestContext,
-  );
+  if (config.chainConfig[chainId].routerContractAddress) {
+    if (isChainSupportedByGelato(chainId)) {
+      try {
+        await gelatoSend(
+          chainId,
+          nxtpContractAddress,
+          encodedData,
+          chainId === txData.sendingChainId ? txData.sendingAssetId : txData.receivingAssetId,
+          "0",
+        );
+      } catch (err) {}
```

Comment:
We should have a general enclosure for try catch that resorts to using txservice as a backup